url.go

```diff
@@ -21,13 +21,10 @@ import (
 type Schema string
 
 const (
-	// CharmStore schema represents the original schema for a charm URL
-	CharmStore Schema = "cs"
-
 	// Local represents a local charm URL, describes as a file system path.
 	Local Schema = "local"
 
-	// CharmHub schema represents the new schema for another unique charm store.
+	// CharmHub schema represents the charmhub charm repository.
 	CharmHub Schema = "ch"
 
 	// HTTP refers to the HTTP schema that is used for the V2 of the charm URL.
```

Comment:
Remove HTTP and HTTPS. It never escapes into juju itself (HTTP(S) is converted into cs URLS), so it makes sense to nuke it here as well.